daisy/compute/compute.go

```diff
@@ -346,7 +346,7 @@ func (c *client) operationsWaitHelper(project, name string, getOperation operati
 
 		switch op.Status {
 		case "PENDING", "RUNNING":
-			time.Sleep(1 * time.Second)
+			time.Sleep(5 * time.Second)
```

Comment:
Maybe make this configurable so we keep the existing behavior where existing errors will retry in 1 second while GOAWAY will retry after 5 seconds.

Reply:
That makes sense. I moved this logic into the should retry function so that specific response doubles the sleep duration multiplier, which is based on the retry attempt count.